packages/cmrjs/src/cmr-utils.js

```diff
@@ -931,6 +931,19 @@ async function getCollectionsByShortNameAndVersion(results) {
   return response.body;
 }
 
+/**
+ * Call CMR to get a list of bucket/paths that a user has access to
+ *
+ * @param {string} edlUser - users earthdata login name
+ * @param {string} cmrProvider - cumulus CMR provider name
+ * @returns {Object} list of bucket/paths that a user has access to
+ */
+async function getUserAccessibleBuckets(edlUser, cmrProvider = process.env.cmr_provider) {
+  const searchParams = { user_id: edlUser, 'provider[]': cmrProvider };
```

Comment:
No, but it can be an array on their endpoint, we are only passing in a single string of cmrProvider.  but you have to use the string as a query param on the `provider[]` value.